client/www/pages/docs/schema.md

```diff
@@ -40,10 +40,18 @@ The first parameter to `i.graph` is a dictionary of entities, where the key repr
 
 Entity definitions accept a map of attribute definitions, where the key represents the attribute name and the value contains configuration for the attribute.
 
-First we specify the expected type of the attribute: `i.string()`, `i.number()`, `i.boolean()`, `i.json()` and `i.any()`.
+First we specify the expected type of the attribute: `i.string()`, `i.number()`, `i.boolean()`, `i.date()`, `i.json()`, or `i.any()`.
+
+{% callout %}
+
+`i.date()` accepts dates as either a numeric timestamp (in milliseconds), or an ISO 8601 string.
```

Comment:
nit: maybe we should give examples of this, as i.m.o most folks don't know what ISO 8601 is. Perhaps if they know it's like `date.toString()`, they'll get it

Reply:
Updated!